apps/condo/domains/ticket/utils/handlers.js

```diff
@@ -78,6 +78,16 @@ const detectEventTypes = ({ operation, existingItem, updatedItem }) => {
     return result
 }
 
+const getResident = async (context, userId, propertyId, organizationId ) => {
```

Comment:
it's better to just call `Resident.getOne` rather than extract it to the function.

Reply:
it's used twice by now, and will be used lot's more in near future. this way it looks a bit shorter

Reply:
It’s ok to call it twice. It’s not looks like a helper. It’s just wrapper around arguments. It’s better just call the original function rather than create logic less wrapper

Reply:
fixed